### ISSUE TITLE
FBC support in Makefile targets 

### ISSUE THREAD

**Issue by Nilesh20** [operator-framework/operator-sdk#6468]
#### What did you do?
We have operator and as a part of it we have auto generated Makefile for bundle generation and catalog image creation & push. 

#### What did you expect to see?

As a part of Makefile I saw that there is no task for catalog creation in auto generated Makefile. So how we can start using FBC in that case ? whether operator-sdk going to add task for catalog creation in Makefile ?

#### Environment

**Operator type:**

**Kubernetes cluster type:**

`$ operator-sdk version` 1.28.0

`$ go version` (if language is Go) 1.20

`$ kubectl version`

**Comment by rashmigottipati** [comment]
@Nilesh20 We are not ready to fully transition to File-Based Catalog (FBC) images yet, although the current Makefile still supports catalog creation for SQLite based index images. 

Please check the similar issue that is open and blocked by the [comment](https://github.com/operator-framework/operator-sdk/issues/5832#issuecomment-1377612560). Closing this as this is a duplicate of [issue](https://github.com/operator-framework/operator-sdk/issues/5832).
